components/NavigationDrawer/tests/unit/MDCNavigationDrawerScrollViewTests.m

```diff
@@ -768,4 +770,66 @@ - (void)testFullScreenContentLayoutCalculationsComplete {
   // It has no visible side effects, so as long as this test finishes, it passes
 }
 
+- (void)testSettingElevationInDrawerViewControllerPropogatesToPresentationController {
+  // Given
+  self.drawerViewController.elevation = 20;
+
+  // Then
+  MDCBottomDrawerPresentationController *presentationController =
+      (MDCBottomDrawerPresentationController *)self.drawerViewController.presentationController;
+  XCTAssertEqualWithAccuracy(presentationController.elevation, 20, 0.001);
+}
+
+- (void)testCorrectDefaultElevationValue {
```

Comment:
Can you add tests that changing the elevation on the presentation controller after presentation will propagate correctly? I saw you added code that modifies the elevation and shadow after presentation.

Reply:
Done.